build-logic/convention/src/main/kotlin/JavaLibrary.kt

```diff
@@ -0,0 +1,6 @@
+import org.gradle.api.JavaVersion
+
+object JavaLibrary {
+  val SOURCE_COMPATIBILITY_JAVA_VERSION = JavaVersion.VERSION_17
```

Comment:
these are used in 1 place only.
Lets set those values directly and get rid of this file :-)

Reply:
Fixed